src/main/drivers/pwm_output.h

```diff
@@ -27,6 +27,8 @@ typedef enum {
     PWM_TYPE_ONESHOT42,
     PWM_TYPE_MULTISHOT,
     PWM_TYPE_BRUSHED,
+    PWM_TYPE_DSHOT1200,
```

Comment:
Sorry to be a pain, but can we have these in ascending order? Ie 
```
typedef enum {
    PWM_TYPE_STANDARD = 0,
    PWM_TYPE_ONESHOT125,
    PWM_TYPE_ONESHOT42,
    PWM_TYPE_MULTISHOT,
    PWM_TYPE_BRUSHED,
    PWM_TYPE_DSHOT150,
    PWM_TYPE_DSHOT300,
    PWM_TYPE_DSHOT600,
    PWM_TYPE_DSHOT900,
    PWM_TYPE_DSHOT1200,
    PWM_TYPE_MAX
} motorPwmProtocolTypes_e;
```
that follows the convention as per the "shot" protocols which has slowest first. It also means we could add a DSHOT1500 in future while preserving the order (unlikely, I know).